packages/graphql-yoga/src/plugins/resultProcessor/sse.ts

```diff
@@ -0,0 +1,88 @@
+import { ExecutionResult } from 'graphql'
+import { isAsyncIterable } from '@envelop/core'
+
+import { getResponseInitByRespectingErrors } from '../../error.js'
+import { FetchAPI, MaybeArray } from '../../types.js'
+import { ResultProcessor, ResultProcessorInput } from '../types.js'
+import { jsonStringifyResult } from './stringify.js'
+
+export interface SSEProcessorOptions {
+  graphqlSSEDistinctConnections?: boolean
```

Comment:
imo we should not prefix `graphqlsse`
```suggestion
  distinctConnections?: boolean
```

Reply:
But it enables support for GraphQL SSE's distinct connections mode.